src/styles/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  display: flex;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__list-el {
+    display: flex;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list-link {
+    line-height: 60px;
+    text-decoration: none;
+    color: var(--main-letters-color);
+    position: relative;
+    font-weight: 500;
+    transition-duration: 300ms;
+
+    &:hover {
+      transition-duration: 300ms;
+      color: var(--button-color);
+    }
+  }
+}
+  
+.is-active {
+  color: var(--button-color);
+
+  &::after {
```

Comment:
The ::after pseudo-element is not correctly positioned. The 'top', 'left', 'right', 'bottom' properties are being used together which can cause unpredictable behavior. It would be better to only specify two values for horizontal and vertical positioning.